molecule/default/converge.yml

```diff
@@ -11,6 +11,6 @@
         express_system_user_home: "/home/{{ express_system_user }}"
         express_node_version: 10.x
         express_git_url: "https://github.com/onaio/reveal-frontend"
-        express_git_version: "tiny-backend"
+        express_git_version: "master"
```

Comment:
I think that it is better to use a specific commit hash

Reply:
ok

Reply:
Or a release tag - could also work

Reply:
Yeah, i will go with the release tag, I agree its actually better.